niworkflows/viz/utils.py

```diff
@@ -174,7 +174,7 @@ def cuts_from_bbox(mask_nii, cuts=3):
         smin, smax = (0, mask_data.shape[ax] - 1)
 
         B = np.argwhere(c > th)
-        if not B.size:  # Threshold too high
+        if not B.size or B.size < cuts:  # Threshold too high
```

Comment:
`B.size` is always numeric:

```suggestion
        if B.size < cuts:  # Threshold too high
```